src/vsphere_cpi/spec/integration/vm_host_rule_spec.rb

```diff
@@ -465,14 +465,14 @@
         context 'and one of the cluster has host group and other has resource pool' do
           let(:options) do
             cpi_options(
-                datacenters: [{
-                                  'datastore_pattern' => @datastore_shared_pattern,
-                                  'persistent_datastore_pattern' => @datastore_shared_pattern,
-                                  clusters: [
-                                      { @cluster_name => { host_group: {'name' => @second_host_group, 'drs_rule' => 'Must'} } },
-                                      { @second_cluster_name => { resource_pool: @second_cluster_resource_pool_name, } },
-                                  ]
-                              }]
+              datacenters: [{
+                              'datastore_pattern' => @datastore_shared_pattern,
+                              'persistent_datastore_pattern' => @datastore_shared_pattern,
+                              clusters: [
+                                { @cluster_name => { resource_pool: @first_cluster_second_resource_pool_name, } },
```

Comment:
this is probably fine, my only worry is that its not exercising the nested resource group code path